custom/icds_reports/tasks.py

```diff
@@ -385,7 +385,7 @@ def _create_aggregate_functions(cursor):
         raise
 
 
-def _update_aggregate_locations_tables():
+def _update_aggregate_locations_tables(agg_date):
```

Comment:
we arent good about this, but it would be good to remove the `_` at the start of the function name since that is supposed to indicated the function is prviate, and this isnt